src/utils/RAML.js

```diff
@@ -184,6 +184,9 @@ export default class RAML {
                             if (methodClassFunction === 'fetch') {
                                 debug(`Special case, this is a POST or PATCH at {id}, which is an update()`);
                                 methodClassFunction = 'update';
+                            } else if (methodClassFunction === 'list' && method.method === 'post') {
```

Comment:
You can remove the `&& method.method === 'post'` as this is caught by the switch statement we are in.